app/Http/Middleware/IsAdmin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace REBELinBLUE\Deployer\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+
+class IsAdmin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        // If user is authenticated ...
+        if (Auth::user()) {
```

Comment:
Take a look at the Authenticate middle, I am injecting the AuthFactory as I really don't like Laravel's "facades"

Reply:
I will check this in the morning